lib/OpenQA/Shared/Plugin/SharedHelpers.pm

```diff
@@ -40,44 +37,38 @@ sub _determine_os_autoinst_web_socket_url {
     return "ws://$host:$port/$job_token/ws";
 }
 
-sub _find_current_job {
-    my $c = shift;
-
+sub _find_current_job ($c) {
     return undef unless my $test_id = $c->param('testid');
     return $c->helpers->schema->resultset('Jobs')->find($test_id);
 }
 
-sub _current_user {
-    my $c = shift;
-
+sub _current_user ($c) {
     # If the value is not in the stash
     my $current_user = $c->stash('current_user');
     unless ($current_user && ($current_user->{no_user} || defined $current_user->{user})) {
         my $id = $c->session->{user};
         my $user = $id ? $c->schema->resultset("Users")->find({username => $id}) : undef;
         $c->stash(current_user => $current_user = $user ? {user => $user} : {no_user => 1});
     }
+    use Data::Dumper;
+    $Data::Dumper::Maxdepth = 2;
```

Comment:
Please don't set global variables. Use `local` instead.

Reply:
that shouldn't even be included, just for debugging, sorry